README.md

```diff
@@ -22,6 +25,29 @@ Hazelcast Command Line is a tool which allows users to install & run Hazelcast I
 - This tool runs under Unix-like environments only.
 - JRE 8+ should be installed.
 
+## Installation
+
+You can install Hazelcast Command Line using [Homebrew](https://brew.sh/) on macOS or manually on other operating systems.
```

Comment:
I would just like to add that the options for other package managers are coming soon. Not to scare the first users.